packages/template-retail-react-app/app/components/store-locator/store-locator-content.jsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import React from 'react'
+import {useIntl} from 'react-intl'
+import PropTypes from 'prop-types'
+import {
+    Heading,
+    Button,
+    InputGroup,
+    Accordion,
+    AccordionItem,
+    Select,
+    Box,
+    Input
+} from '@salesforce/retail-react-app/app/components/shared/ui'
+import StoresList from '@salesforce/retail-react-app/app/components/store-locator/stores-list'
+import {Controller, useForm} from 'react-hook-form'
+import {SUPPORTED_STORE_LOCATOR_COUNTRIES} from '@salesforce/retail-react-app/app/constants'
+
+const StoreLocatorContent = (props) => {
+    const {searchStoresData, searchStoresParams, setSearchStoresParams} = props
+
+    const intl = useIntl()
+    const formProps = useForm({
+        mode: 'onChange',
+        reValidateMode: 'onChange',
+        defaultValues: {
+            countryCode: searchStoresParams?.countryCode,
+            postalCode: searchStoresParams?.postalCode
+        }
+    })
+    const {control, handleSubmit} = formProps
+
+    return (
+        <>
+            <Heading fontSize="2xl" style={{marginBottom: '25px'}}>
+                {intl.formatMessage({
+                    id: 'store_locator.title',
+                    defaultMessage: 'Find a Store'
+                })}
+            </Heading>
+            <InputGroup>
+                <Controller
+                    name="postalCode"
+                    control={control}
+                    defaultValue={searchStoresParams?.postalCode}
+                    render={({field}) => {
+                        return <Input {...field} marginBottom="10px" />
+                    }}
+                ></Controller>
```

Comment:
Would you consider using a placeholder attribute here to make it clear that this value is intended to be Zip code?

If you remove the default text it isn't entirely clear:

![Screenshot 2024-06-13 at 3 23 00 PM](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/28967/77d244f3-018d-4ecc-aa87-99110c10e0e1)